api/post.go

```diff
@@ -405,6 +405,31 @@ func fireAndForgetNotifications(post *model.Post, teamId, teamUrl string) {
 					bodyPage.Props["PostMessage"] = model.ClearMentionTags(post.Message)
 					bodyPage.Props["TeamLink"] = teamUrl + "/channels/" + channel.Name
 
+					// attempt to fill in a message body based if the message has none
```

Comment:
Oops, I must've got distracted while writing it so it ended up making no sense. I'll fix that.